study-builder/studies/testboston/patches/vaccine-copy-answer-event.conf

```diff
@@ -0,0 +1,21 @@
+{
+  "trigger": {
+    "type": "ACTIVITY_STATUS",
+    "activityCode": ${id.act.longitudinal},
+    "statusType": "CREATED"
+  },
+  "action": {
+    "type": "COPY_ANSWER",
+    "copyFromPreviousInstance": true,
+    "previousInstanceQuestionStableIds": [
+      "LONGITUDINAL_VACCINE_STUDY",
+      "LONGITUDINAL_VACCINE_STUDY_DATE",
+      "LONGITUDINAL_VACCINE_RECEIVED",
+      "LONGITUDINAL_VACCINE_RECEIVED_NAME",
+      "LONGITUDINAL_VACCINE_RECEIVED_DATE"
+    ],
```

Comment:
Is this what will be supported in the new feature?

Reply:
Yup! It will be supported by new feature added in #584.